packages/orbit-components/src/Modal/ModalSection/index.tsx

```diff
@@ -29,7 +29,7 @@ export const ModalSectionWrapper = ({
         "orbit-modal-section",
         "py-lg px-md lm:p-xl box-border w-full",
         "border-b-elevation-flat-border-color border-b border-solid",
-        "last-of-type:[&:not(:last-child)]:rounded-b-none",
+        "lm:last-of-type:rounded-b-modal last-of-type:[&:not(:last-child)]:rounded-b-none",
```

Comment:
`lm:last-of-type:rounded-b-modal last-of-type:[&:not(:last-child)]:rounded-b-none`

So that means that:
- on large mobile and last of type, we get rounded-b-modal
- otherwise, last of type and not last child, we get rounded-b-none

because of the order of the classes, that means there would still be situations when the corners wouldn't be rounded on desktop, right?

Reply:
The order of classes is changed automatically in Tailwind (eslint? prettier?) so I'm not sure how they are applied.

I guess specificity plays a role in these classes, and the last-child rule takes precedence, but I'm not fully sure.

Reply:
I can confirm that the `:not(:last-child)` has precedence over only `last-of-type`.

As an example, this is the last section in a modal with a footer:

![image](https://github.com/kiwicom/orbit/assets/8863238/d0b1a267-fb78-4541-91e4-8e0b76f4b2ae)

As one can see, the class with `:not(:last-child)` overrides the other one, generating the desired result: removing the bottom borders when the last section is not the last child of the block.

Reply:
I wonder if it's just because of how Tailwind writes the CSS file 🤔 

For example, the class `lm:last-of-type:rounded-b-modal` is contained in a media query, so maybe Tailwind writes media query-related classes later in the CSS file compared to "normal" classes, and that's why we're lucky in this instance.

I wonder how "class-specificity" works here 😅 🙈 🤔

Reply:
I think this proves my theory, I don't know what else I can try 😅 

[Playground](https://codi.link/PHNlY3Rpb24+CiAgPHNwYW4+c3Bhbjwvc3Bhbj4KICA8ZGl2Pm9uZTwvZGl2PgogIDxkaXY+dHdvPC9kaXY+CiAgPHNwYW4+c3Bhbjwvc3Bhbj4KICA8ZGl2PnRocmVlPC9kaXY+CiAgPGRpdj5mb3VyPC9kaXY+Cjwvc2VjdGlvbj4=%7CZGl2Om5vdCg6Zmlyc3QtY2hpbGQpOmZpcnN0LW9mLXR5cGUgewogIGNvbG9yOiB0b21hdG8KfQoKZGl2OmZpcnN0LW9mLXR5cGUgewogIGNvbG9yOiBwdXJwbGUKfQoK%7C)